### ISSUE TITLE
Make phpinsights check tests directory

### ISSUE THREAD

**Issue by alexmakii** [nunomaduro/phpinsights#553]
| Q                | A
| ---------------- | -----
| Bug report?      | no
| Feature request? | yes
| Library version  | 1.14 ... 2.0

phpinsights ignores `tests/` directory.

Documentation states that by default it should exclude only `vendor`, `node_modules` and `bower_components`.

I agree that fixing issues in tests can be cumbersome. But phpinsights helps keeping same code style for all tracked files and tests aren't exclusion.
Currently I can run phpinsights with explicit check of `tests` folder but:
- that's unexpected behavior (found when my teammate added obviously bad code in it's PR and pipelines were not failed)
- hards manual use of phpinsights - you need to keep that in mind and manually run phpinsights multiple times (if there are not tools)
- pipelines needs to run more steps (that can be confusing).

Can we remove exclusion of `tests` or make it configurable with default set to true (excluded) ?

I'll create a PR if any of above is acceptable.

```yaml
# phpinsights.php
return [

    'preset' => 'default',

    'requirements' => [
        'min-architecture' => 100,
        'min-quality'      => 100,
        'min-complexity'   => 85,
        'min-style'        => 100,
    ],

    'exclude' => [
        'resources/',
        'storage/',
    ],
];
```

**Comment by JustSteveKing** [comment]
Will investigate this one.

@alexmakii sorry new maintainers taking over so making notes. Is this still an issue for you?

**Comment by alexmakii** [comment]
@JustSteveKing 
Yep, it still an issue. I've overpass this by adding additional step in pipelines, but...
It's an unexpected behavior, and I found this issue accidentally seen that my teammate added the obviously poorly written code in a test class.

As I understand, best option will be to add `check_tests: true` or `add_tests: true` till next major release. And make it default behavior on major release (removing the option)

P.S. My company uses lumen and we've decided to check all files created by developers (except storage/resources and blade templates). As result we found mediocre number of code flaws.
